daemon/job_actual_ip.go

```diff
@@ -0,0 +1,67 @@
+package daemon
+
+import (
+	"context"
+	"log"
+	"net/netip"
+	"time"
+
+	"github.com/NordSecurity/nordvpn-linux/core"
+	"github.com/NordSecurity/nordvpn-linux/internal"
+	"github.com/NordSecurity/nordvpn-linux/network"
+)
+
+func insightsUntilSuccess(ctx context.Context, api core.InsightsAPI) (core.Insights, error) {
+	for i := 0; ; i++ {
```

Comment:
Is there a reason that this is infinite? Shouldn't we have some max retries?
Also - if the `api.Insights()` will keep failing, we'll spam logs, with the error, indefinitely